configuration.json

```diff
@@ -2,14 +2,108 @@
   "videoRecord": false,
   "videoDir": "build/cify/videos/",
   "capabilities": {
-    "android": {
-      "capability": "android"
-    },
-    "browser": {
-      "capability": "chrome"
-    },
-    "ios": {
-      "capability": "iphone"
-    }
+    "browser": [
```

Comment:
As I understand - it's used for tests only, correct? If yes - please move it from root folder under tests.

Reply:
disagree, for now let's leave it as is, this would bing about too many changes and is not part of this task

Reply:
Yep, done.